### PR TITLE
Add support for eth-secp256k1 addresses for ethermint

### PR DESCRIPTION
**Pull request by devashishdxt**

### CODE REVIEW

**Review by tomtau — COMMENTED** [review]
looks all right -- given at least the protobufs were taken from an external repo, perhaps add a NOTICE file + copyright info?

**Comment by devashishdxt** [comment]
> looks all right -- given at least the protobufs were taken from an external repo, perhaps add a NOTICE file + copyright info?

Yes. I'll add for all the external proto files used.

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Crypto.com Chain
```
Comment (tomtau) [inline NOTICE:1]:
```suggestion
IBC Solo Machine
```

File: NOTICE [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ * Copyright: Copyright (c) 2016-2020 All in Bits, Inc
+ * License: Apache License 2.0
+ * Repository: https://github.com/cosmos/cosmos-sdk
+
+* IBC
+ * Copyright: Copyright (c) 2021 COSMOS
+ * License: MIT License
+ * Repository: https://github.com/cosmos/ibc-go
+
+* Ehtermint
```
Comment (tomtau) [inline NOTICE:16]:
```* Ethermint
 * Copyright: Copyright (c) 2016-2021 Ethermint contributors
```

File: solo-machine-core/proto/ethermint/crypto/v1alpha1/ethsecp256k1/keys.proto [hunk 3]
```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
```
Comment (tomtau) [inline solo-machine-core/proto/ethermint/crypto/v1alpha1/ethsecp256k1/keys.proto:1]: perhaps copyright headers in the proto files?

**Review by tomtau — APPROVED** [review]
👌

**Pull request merged**